src/app/containers/Image/index.stories.jsx

```diff
@@ -1,7 +1,7 @@
 import { storiesOf } from '@storybook/react';
 import React from 'react';
 import ImageContainer from '.';
-import { GhostWrapper } from '../../lib/styledGrid';
+import { GhostGrid } from '../../lib/styledGrid';
```

Comment:
Could you please use `Grid` in this file? This is a story outside of the Article, so it can have a grid without a background colour.

Reply:
done :)